test/mocha/assertions.js

```diff
@@ -0,0 +1,170 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+
+export const shouldNotError = ({response, error, data, path}) => {
```

Comment:
We usually use `export function shouldNotError` style here, not arrow functions.

```suggestion
export const shouldNotError = ({response, error, data, path}) => {
```

Reply:
made the changes from arrows to function here: https://github.com/digitalbazaar/bedrock-vc-exchanger/pull/10/commits/f94cffa35b188259a32394663c082588149327e6